.circleci/main.yml

```diff
@@ -7,6 +7,11 @@ aliases:
       ignore: /.*/
     tags:
       only: /^v\d+\.\d+\.\d+(-[0-9A-Za-z-]+(\.[0-9A-Za-z-]+)*)?$/
+  - &only-nightly-tags
+    branches:
+      ignore: /.*/
```

Comment:
Should this explicitly select master?  It would be annoying if someone tagged the 2.3.x branch and we got a 2.4.x nightly (which is hard-coded later).

Reply:
good catch, i have locked it to only master